pkgs/desktops/gnome-3/apps/ocrfeeder/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, fetchurl
+, pkgconfig
+, gtk3
+, gtkspell3
+, isocodes
+, goocanvas2
+, intltool
+, itstool
+, libxml2
+, gnome3
+, python3
+, gobject-introspection
+, wrapGAppsHook
+, tesseract4
+, extraOcrEngines ? [] # other supported engines are: ocrad gocr cuneiform
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Since this is a Python app, it might make sense to use `buildPythonApplication`.

Reply:
There is no setup.py there so it seems to make more sense to use `mkDerivation`.